src/js/components/Calendar/Calendar.js

```diff
@@ -82,15 +82,90 @@ const buildDisplayBounds = (reference, firstDayOfWeek) => {
 
 const millisecondsPerYear = 31557600000;
 
+const CalendarDayButton = ({
+  a11yTitle,
+  active,
+  disabled,
+  fill,
+  onClick,
+  onMouseOver,
+  onMouseOut,
+  children,
+}) => {
+  return (
+    <Button
+      a11yTitle={a11yTitle}
+      tabIndex={-1}
+      active={active}
+      disabled={disabled}
+      onClick={onClick}
+      onMouseOver={onMouseOver}
+      onMouseOut={onMouseOut}
+      onFocus={() => {}}
+      onBlur={() => {}}
+      fill={fill}
+      plain
+    >
+      {children}
+    </Button>
+  );
+};
+
+const CalendarDay = ({
+  children,
+  fill,
+  size,
+  isInRange,
+  isSelected,
+  otherMonth,
+  buttonProps = {},
+}) => {
+  return (
+    <StyledDayContainer sizeProp={size} fillContainer={fill}>
+      <CalendarDayButton fill={fill} {...buttonProps}>
+        <StyledDay
+          inRange={isInRange}
+          otherMonth={otherMonth}
+          isSelected={isSelected}
+          sizeProp={size}
+          fillContainer={fill}
+        >
+          {children}
+        </StyledDay>
+      </CalendarDayButton>
+    </StyledDayContainer>
+  );
+};
+
+const CalendarCustomDay = ({ children, fill, size, buttonProps }) => {
+  if (!buttonProps) {
+    return (
+      <StyledDayContainer sizeProp={size} fillContainer={fill}>
+        {children}
+      </StyledDayContainer>
+    );
+  }
+
+  return (
+    <StyledDayContainer sizeProp={size} fillContainer={fill}>
+      <CalendarDayButton fill={fill} {...buttonProps}>
+        {children}
+      </CalendarDayButton>
+    </StyledDayContainer>
+  );
+};
+
 const Calendar = forwardRef(
   (
     {
       animate = true,
       bounds: validBounds,
+      children,
       date: dateProp,
       dates: datesProp,
       daysOfWeek,
       disabled,
+      fill,
```

Comment:
We need to update Calendar/doc.js and Calendar/index.d.ts for `fill` and `children`.

Reply:
Updates made.